src/Microsoft.OpenApi/Services/OpenApiUrlTreeNode.cs

```diff
@@ -150,6 +150,13 @@ public void Attach(OpenApiDocument doc, string label)
             }
 
             var segments = path.Split('/');
+            if (path.EndsWith("/"))
```

Comment:
```suggestion
            if (path.EndsWith('/'))
```

Reply:
.NET Standard 2.0 doesn't support the `EndsWith(Char)` overload, see <https://learn.microsoft.com/dotnet/api/system.string.endswith?view=netstandard-2.0>.

Reply:
thanks, I forgot we had such a low compatibility bar.... Can you pass StringComparison.OrdinalIgnoreCase then please? Won't make a perf difference, but it will avoid triggering static analysis warnings if we add some more in the future.

Reply:
Done, thanks.